backend/src/middleware/hashtags/hashtagsMiddleware.spec.js

```diff
@@ -69,8 +69,27 @@ afterEach(async () => {
 describe('hashtags', () => {
   const id = 'p135'
   const title = 'Two Hashtags'
-  const postContent =
-    '<p>Hey Dude, <a class="hashtag" href="/search/hashtag/Democracy">#Democracy</a> should work equal for everybody!? That seems to be the only way to have equal <a class="hashtag" href="/search/hashtag/Liberty">#Liberty</a> for everyone.</p>'
+  const postContent = `
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-13T17:50:00Z" title="Friday, September 13th 2019, 7:50:00 pm +02:00">Sep 13, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -129,8 +129,8 @@ describe('hashtags', () => {
         )
       })
 
-      describe('afterwards update the Post by removing a Hashtag, leaving a Hashtag and add a Hashtag', () => {
-        // The already existing Hashtag has no class at this point.
+      describe('afterwards update the Post by removing a hashtag, leaving a hashtag and add a hashtag', () => {
```


I'm sorry, but it's not clear what you are trying to say here... I know you are just changing the case here, but still... not sure what you mean
maybe you could explain it, and I can help improve the English translation?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-14T11:54:18Z" title="Saturday, September 14th 2019, 1:54:18 pm +02:00">Sep 14, 2019</time>_

---


I my self had to read it twice, but than understand.
I should have corrected this at that moment, but somehow got over it. @mattwr18 

There is already a post generated with #Democracy and #Liberty hashtags.
By updating the post with #Elections and #Liberty only this both should be related to the post. So the statement was one (#Democracy) is removed, one is left (#Liberty) and one is new (#Elections). Much to complicated …
But a statement like `only the both hashtags in the updated content are related` will be clear and much more understandable.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-16T08:32:44Z" title="Monday, September 16th 2019, 10:32:44 am +02:00">Sep 16, 2019</time>_

---


```suggestion
      describe('updates the Post by removing, keeping and adding one hashtag respectively', () => {
```